symengine/catch/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(catch)
+
+set(SRC
+    catch.cpp
+)
+
+add_library(catch STATIC ${SRC})
+target_link_libraries(catch symengine)
+
+install(TARGETS catch
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+        )
```

Comment:
So can these 4 lines can be removed?
I just followed `Teuchos` on T. Does `Teuchos` needs to be installed?